base/iostream.jl

```diff
@@ -226,6 +265,15 @@ function readbytes_some!(s::IOStream, b::Array{UInt8}, nb)
     return nr
 end
 
+"""
+    readbytes!(stream::IOStream, b::AbstractVector{UInt8}, nb=length(b); all::Bool=true)
+
+Read at most `nb` bytes from `stream` into `b`, returning the number of bytes read.
+The size of `b` will be increased if needed (i.e. if `nb` is greater than `length(b)`
+and enough bytes could be read), but it will never be decreased.
+
+See [`read`](:func:`read`) for a description of the `all` option.
+"""
```

Comment:
Is this one not in the rst any more though? looks like the line ```See ``read`` for a description of the ``all`` option.``` is being deleted?